src/main/java/ui/Ui.java

```diff
@@ -1,52 +1,104 @@
 package ui;
 
+import language.Language;
+import language.LanguageFactory;
+import language.Languages;
 import move.Moves;
 
 import java.io.InputStream;
 import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
 
 public class Ui {
 
+    final static private String NEWLINE = "";
```

Comment:
This variable is called NEWLINE but is an empty string? Did you mean for this to be `\n`?

Reply:
You're right!! That doesn't make sense...